sqrl-cli/src/main/java/com/datasqrl/cli/output/TestOutputManager.java

```diff
@@ -31,30 +32,31 @@ public class TestOutputManager implements AutoCloseable {
 
   private final Path rootDir;
 
-  private PrintStream originalOut;
-  private PrintStream originalErr;
+  private final PrintStream originalOut = System.out;
+  private final PrintStream originalErr = System.err;
   private PrintStream logStream;
+  private ErrorCapturingStream errorCapturingStream;
 
-  public void redirectStd() {
-    originalOut = System.out;
-    originalErr = System.err;
+  @SneakyThrows
+  public void init() {
+    var logsDir = rootDir.resolve("build/logs");
+    Files.createDirectories(logsDir);
 
-    initLogStream();
+    var logFile = logsDir.resolve(TEST_LOG_FILE).toFile();
+    var fileOutputStream = new FileOutputStream(logFile, true);
+    errorCapturingStream = new ErrorCapturingStream(fileOutputStream);
+    logStream = new PrintStream(errorCapturingStream);
 
     System.setOut(logStream);
     System.setErr(logStream);
   }
 
-  public void restoreStd() {
-    if (originalOut != null) {
-      System.setOut(originalOut);
-      originalOut = null;
-    }
+  public PrintStream getOriginalOut() {
+    return originalOut;
+  }
 
-    if (originalErr != null) {
-      System.setErr(originalErr);
-      originalErr = null;
-    }
+  public PrintStream getOriginalErr() {
+    return originalErr;
   }
```

Comment:
These getter are unnecessary, see my other comment about it on the `TestCmd` file.